src/Microsoft.Azure.WebJobs.Host/Scale/ScaleLoggerExtension.cs

```diff
@@ -49,6 +50,12 @@ public static partial class ScaleLoggerExtension
                 FunctionScaleErrorEventId,
                 "Function '{functionName}' error: {message}");
 
+        private static readonly Action<ILogger, string, string, Exception> _logFunctionScaleWarning =
+            LoggerMessage.Define<string, string>(
+                LogLevel.Warning,
+                FunctionScaleWarningEventId,
+                "Function '{functionName}' error: {message}");
```

Comment:
Should this be `warning` not `error`?

```suggestion
                "Function '{functionName}' warning: {message}");
```